godot-core/src/docs.rs

```diff
@@ -79,20 +95,33 @@ pub fn gather_xml_docs() -> impl Iterator<Item = String> {
 
             let InherentImplDocs {
                 methods,
-                signals,
-                constants,
+                signals_block,
+                constants_block,
             } = pieces.inherent;
 
             let virtual_methods = pieces.virtual_methods;
-            let brief = description.split_once("[br]").map(|(x, _)| x).unwrap_or_default();
+            let methods_block = if methods.is_some() || virtual_methods.is_some() {
+                format!(
+                    "<methods>{m}{vm}</methods>",
+                    m=methods.unwrap_or_default(),
+                    vm=virtual_methods.unwrap_or_default())
```

Comment:
```suggestion
                    "<methods>{m}{vm}</methods>",
                    m = methods.unwrap_or_default(),
                    vm = virtual_methods.unwrap_or_default())
```
I missed this, sorry.